src/firefly/java/edu/caltech/ipac/table/io/IpacTableWriter.java

```diff
@@ -94,7 +94,7 @@ private static void save(PrintWriter out, DataGroup dataGroup) throws IOExceptio
                 IpacTableUtil.writeAttributes(out, atts, false);
 
                 dt.setDataType(String.class);
-                dt.setTypeDesc(DataType.CHAR);
+                dt.setTypeDesc("char");
```

Comment:
Why not a constant?

Reply:
In the past, Firefly mimic data type of IPAC table.  That's why they are used interchangeably.  I just wanted to be clear that this class is for IPAC table format.  Its type reflect IPAC table's.  I can declare a const here, but it seem a bit redundant since it won't use anywhere else.